### DISCUSSION TITLE
Prefix for `no_results` in the Query module

### DISCUSSION THREAD

**Discussion by shbchk** [ExpressionEngine/ExpressionEngine#4640] [Feature Requests]
### Suggest an idea for ExpressionEngine

Currently, it is impossible to use the Query module inside a `{exp:channel:entries}` tag - `{if no_results}` inside the `{exp:query}` will be ignored.

Sure, there are embeds and pro_variables available to solve this, but still, it would be great to have a prefix.

**Comment by intoeetive** [comment]
It's not specific to Query. Whenever you are nesting tags and using no_results, you need to use it for each pair of nested tags
